connectors/source.py

```diff
@@ -235,3 +235,11 @@ def get_data_sources(config):
     """Returns an iterator of all registered sources."""
     for name, fqn in config["sources"].items():
         yield get_source_klass(fqn)
+
+
+def get_data_source_dict(config):
```

Comment:
I would not use data type in the functions` names. Does it technically return data source configuration?

Reply:
No, it returns a service type - source class dictionary. I just followed the pattern of the above method `get_data_sources`, and now I reallized it was named wrongly. Probably we should rename the above method to `get_source_klasses` and this method to `get_source_klass_dict`

Reply:
What is `source class dictionary`?

Reply:
I see it's just a "string" -> "string" dictionary, key is service_type, value is class FQN (see config.yml's "sources" section: https://github.com/elastic/connectors-python/blob/main/config.yml#L38)

Reply:
It's not a `string` (service type) -> `string` (FQN) dictionary. It's a `string` (service type) -> `class` (child classes of `BaseDataSource`). 

It serve exactly the same purpose as the [REGISTRY](https://github.com/elastic/connectors-ruby/blob/9d5be7a79ee18c794403d14b1dcfc2c574786771/lib/connectors/registry.rb#L40) of `connectors-ruby`